.github/workflows/check-package.yml

```diff
@@ -41,9 +41,6 @@ jobs:
 
       - name: Check local package
         run: |
-          pip install -q -U check-manifest
-          # check MANIFEST.in
-          check-manifest
```

Comment:
Removed check-manifest use as it contradicts the output of bdist_wheel

Same as done for PL in in https://github.com/Lightning-AI/lightning/pull/7614